docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -254,6 +254,22 @@ For Oracle ``NUMBER`` (without precision and scale), you can change
 
 .. _datetime mapping:
 
+.. _sqlserver-sql-support:
```

Comment:
should be oracle, not sqlserver

Reply:
I corrected this while you were reviewing. Already amended.